src/installing-workflow/configuring-registry.md

```diff
@@ -0,0 +1,56 @@
+# Configuring Registry
+
+Deis Workflow's builder component relies on registry for storing the application docker images.
+
+Deis Workflow ships with [registry][registry] component by default, which provides in-cluster docker registry backed by the platform configured [object storage][storage]. Few operators might want to use off-cluster registry mainly for performance or security reasons.
+
+## Configuring off-cluster Private Registry
+
+Every component that relies on registry uses two inputs for configuration:
+
+1. Registry Location environment variable named `DEIS_REGISTRY_LOCATION`
+2. Access credentials stored as a Kubernetes secret named `registry-secret`
+
+The helm classic chart for Deis Workflow can be easily configured to connect Workflow components to off-cluster registry. Deis Workflow currently supports Google Container Registry, Amazon ECR and other off-cluster storage providers like dockerhub, quay.io, etc., which use long lived authentication details. If using Google Container Registry or Amazon ECR, Deis Workflow will generate short lived token for authenticating with the registry such that the deployed apps will only have access to the short lived tokens and not to the actual credentials.
```

Comment:
This paragraph needs some work (see below).

Would be good to explain the difference between long lived tokens (lets call it tokens and not details) and short lived tokens, you can do that while talking about GCR and ECR.

Not sure you need to call out that deployed app will not have access to the tokens, I don't see how they would acquire access to those details either way